src/pages/ProductDetail/ProductDetail.jsx

```diff
@@ -0,0 +1,237 @@
+                    >
+                      취소
+                    </button>
+                    <button
+                      type="button"
+                      onClick={() => handleUpdateComment(comment.id)}
+                      className={styles.saveButton}
+                    >
+                      수정 완료
+                    </button>
+                  </div>
+                </div>
+              ) : (
+                <div key={comment.id} className={styles.commentItem}>
+                  <div className={styles.comment}>
+                    <p className={styles.commentText}>{comment.content}</p>
+                    <div className={styles.metaRow}>
+                      <img
+                        src="/profile.svg"
+                        alt={comment.writer.nickname}
+                        className={styles.avatar}
+                      />
+                      <div className={styles.commentContent}>
+                        <p className={styles.nickname}>
+                          {comment.writer.nickname}
+                        </p>
+                        <p className={styles.date}>
+                          {new Date(comment.updatedAt).toLocaleDateString()}
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+                  <div className={styles.menuWrapper}>
+                    <button
+                      type="button"
+                      className={styles.menuButton}
+                      onClick={() =>
+                        setOpenMenuId((prev) =>
+                          prev === comment.id ? null : comment.id
+                        )
+                      }
+                    >
+                      <img
+                        src={kebabIcon}
+                        alt="점 아이콘"
+                        className={styles.kebabIcon}
+                      />
+                    </button>
+
+                    {openMenuId === comment.id && (
+                      <div className={styles.dropdown}>
+                        <button
+                          type="button"
+                          onClick={() => {
+                            setEditingCommentId(comment.id);
+                            setEditingContent(comment.content);
+                            setOpenMenuId(null);
+                          }}
+                        >
+                          수정하기
+                        </button>
+                        <button type="button">삭제하기</button>
+                      </div>
+                    )}
+                  </div>
+                </div>
+              )
+            )}
+          </div>
+        )}
+      </section>
+
+      <button
+        type="button"
+        onClick={() => navigate("/items")}
+        className={styles.backButton}
+      >
+        목록으로 돌아가기
+        <img src={backIcon} alt="돌아가기 아이콘" className={styles.back} />
+      </button>
```

Comment:
💊 제안
지금 코드만으로 보면 버튼 태그보다 Link 태그를 쓰는게 더 적절해보여요.
단순 페이지 이동이 아니라 특정 로직을 실행해야 할때 지금처럼 버튼 태그를 사용하시면 좋을 것 같습니다.
링크 태그를 사용하시면 마우스 오른쪽 클릭이나 새탭 기능도 제공이 가능해서 UX 측면에서도 좋습니다.

```suggestion
      <Link to="/items" className={styles.backButton}>
        목록으로 돌아가기
        <img src={backIcon} alt="돌아가기 아이콘" className={styles.back} />
      </Link>
```